src/LCD_Fonts.h

```diff
@@ -0,0 +1,40 @@
+/**
+ * A library to control the monochrome LCD displays found on Nokia 5110.
+ *
+ * It is a derivative of Jim Lindblom's work, from Sparkfun. The goal
+ * was to decrease its size so it can be ran on ATTiny microcontrollers
+ * that have limited RAM size. Therefore, the main difference is that a
+ * framebuffer, which would consume all dynamic memory, is not used. Get the
+ * original library [here](https://github.com/sparkfun/GraphicLCD_Nokia_5110).
+ *
+ * On how to use this library, please refer to the `examples/` folder.`
+ */
+#pragma once
+
+using GetFontCallback = const unsigned char* (*)(char);
+
+class LcdFont
+{
+public:
+    const static uint8_t ROWS_PER_CHARACTER = 8;
```

Comment:
I'd say remove this field from here completely and use `Nokia_LCD_Fonts::kRows_per_character` instead that already exists.

Reply:
Oh, okay. I got it